### PR TITLE
[close #12] - extract `Sensorario\Application` component

### LINKED ISSUE #12: extract`Sensorario\Applocation` component from web/index.php
- [ ] create new class `Application`
- [ ] inject configuration from `web/index.php`
- [ ] accept symfony like environment  variable

### PR DESCRIPTION
**Pull request by sensorario**
- [x] test coverage

### CODE REVIEW

**Comment by sensorario** [comment]
This PR is related to issue #12

File: src/Sensorario/Application.php [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        ) {
+            throw new \RuntimeException(
+                'Oops! HTTP method not allowed'
+            );
+        }
+
+        return $this->variables;
+    }
+
+    private function isPostRequest()
```
Comment (AlessandroMinoccheri) [inline src/Sensorario/Application.php]: return bool?

File: src/Sensorario/Application.php [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    {
+        if ($this->request->isPostRequest()) {
+            $this->variables = $this->request->getPostVars();
+            return true;
+        }
+
+        return false;
+    }
+
+    private function isGetRequest()
```
Comment (AlessandroMinoccheri) [inline src/Sensorario/Application.php]: return bool?

File: src/Sensorario/Application.php [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+                JSON_PRETTY_PRINT
+            )
+        );
+
+        echo $content;
+
+        exit(0);
+    }
+
+    private function getHttpVerb()
```
Comment (AlessandroMinoccheri) [inline src/Sensorario/Application.php]: return some type?

File: src/Sensorario/Application.php [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+        $this->logger->logResponse(
+            $content = json_encode(
+                $this->config['responses'][$uri],
+                JSON_PRETTY_PRINT
+            )
+        );
+
+        echo $content;
+
+        exit(0);
```
Comment (AlessandroMinoccheri) [inline src/Sensorario/Application.php]: meaning of this?

Comment (sensorario) [inline src/Sensorario/Application.php]: It's a typo due to a refactoring waste code XD.

Comment (sensorario) [inline src/Sensorario/Application.php]: removed!

**Pull request merged**